packages/graphql-docs-generator/src/generator/utils/getSchemaType.ts

```diff
@@ -14,8 +13,6 @@ export const getSchemaType = (schema: string) => {
     }
   }
   catch {
-    return schemaType;
+    return SchemaType.SDL;
   }
-
-  return schemaType;
```

Comment:
There will be no return for when the schema is successfully parsed as JSON but does not have a `data` or `__schema` property. What is correct behavior here? Throw an error or default to SDL?

Reply:
Oh I see now that it throws an error later in `buildSchema`. Maybe add to the docstring that returning `undefined` is the intended behavior (but not a blocker).